lib/rspec/core/formatters/exception_formatter.rb

```diff
@@ -0,0 +1,136 @@
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class ExceptionFormatter
```

Comment:
Yeah I know, but so are the deprecation and bisect formatters etc and they're real formatters. Thanks for making the change though, :heart: